### ISSUE TITLE
Error on 2021.2 EAP versions

### ISSUE THREAD

**Issue by itsxales** [Almighty-Alpaca/JetBrains-Discord-Integration#158]
Discord Integration is not working with the newest Intellij IDEA version.
```java
java.lang.ClassNotFoundException: 
                com.almightyalpaca.jetbrains.plugins.discord.plugin.development.DisableDynamicPluginComponent
             PluginClassLoader(plugin=PluginDescriptor(name=Discord Integration, id=com.almightyalpaca.intellij.plugins.discord, descriptorPath=plugin.xml, path=~\AppData\Local\JetBrains\Toolbox\apps\IDEA-U\ch-0\212.3116.43.plugins\plugin, version=1.6.1, package=null), packagePrefix=null, instanceId=40, state=active)
    at com.intellij.ide.plugins.cl.PluginClassLoader.loadClass(PluginClassLoader.java:215)
    at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:522)
    at com.intellij.serviceContainer.ComponentManagerImpl.registerComponent(ComponentManagerImpl.kt:433)
    at com.intellij.serviceContainer.ComponentManagerImpl.registerComponents(ComponentManagerImpl.kt:372)
    at com.intellij.serviceContainer.ComponentManagerImpl.access$registerComponents(ComponentManagerImpl.kt:57)
    at com.intellij.serviceContainer.ComponentManagerImpl.registerComponents(ComponentManagerImpl.kt:250)
    at com.intellij.idea.ApplicationLoader$initApplication$2$1.accept(ApplicationLoader.kt:75)
    at com.intellij.idea.ApplicationLoader$initApplication$2$1.accept(ApplicationLoader.kt)
    at java.base/java.util.concurrent.CompletableFuture.uniAcceptNow(CompletableFuture.java:753)
    at java.base/java.util.concurrent.CompletableFuture.uniAcceptStage(CompletableFuture.java:731)
    at java.base/java.util.concurrent.CompletableFuture.thenAccept(CompletableFuture.java:2108)
    at java.base/java.util.concurrent.CompletableFuture.thenAccept(CompletableFuture.java:143)
    at com.intellij.idea.ApplicationLoader$initApplication$2.apply(ApplicationLoader.kt:73)
    at com.intellij.idea.ApplicationLoader$initApplication$2.apply(ApplicationLoader.kt)
    at java.base/java.util.concurrent.CompletableFuture$UniCompose.tryFire(CompletableFuture.java:1072)
    at java.base/java.util.concurrent.CompletableFuture$Completion.run(CompletableFuture.java:478)
    at com.intellij.idea.ApplicationLoader$initApplication$3.execute(ApplicationLoader.kt:105)
    at java.base/java.util.concurrent.CompletableFuture.uniComposeStage(CompletableFuture.java:1118)
    at java.base/java.util.concurrent.CompletableFuture.thenComposeAsync(CompletableFuture.java:2246)
    at java.base/java.util.concurrent.CompletableFuture.thenComposeAsync(CompletableFuture.java:143)
    at com.intellij.idea.ApplicationLoader.initApplication(ApplicationLoader.kt:63)
    at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
    at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
    at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
    at java.base/java.lang.reflect.Method.invoke(Method.java:566)
    at com.intellij.idea.a.a(a.java:97)
    at com.intellij.ide.b.o.R.a(R.java:117)
    at com.intellij.ide.b.o.aQ.a(aQ.java:304)
    at com.intellij.ide.b.o.aQ.a(aQ.java:141)
    at com.intellij.ide.b.o.aQ.b(aQ.java:91)
    at java.base/java.util.concurrent.CompletableFuture.uniAcceptNow(CompletableFuture.java:753)
    at java.base/java.util.concurrent.CompletableFuture.uniAcceptStage(CompletableFuture.java:731)
    at java.base/java.util.concurrent.CompletableFuture.thenAccept(CompletableFuture.java:2108)
    at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
    at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
    at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
    at java.base/java.lang.reflect.Method.invoke(Method.java:566)
    at com.intellij.ide.b.o.aQ.a(aQ.java:90)
    at com.intellij.ide.b.j.e.a(e.java:95)
    at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
    at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
    at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
    at java.base/java.lang.reflect.Method.invoke(Method.java:566)
    at com.intellij.idea.MainImpl.start(MainImpl.java:91)
    at com.intellij.idea.StartupUtil.lambda$start$16(StartupUtil.java:275)
    at java.base/java.util.concurrent.CompletableFuture$UniCompose.tryFire(CompletableFuture.java:1072)
    at java.base/java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:506)
    at java.base/java.util.concurrent.CompletableFuture$AsyncSupply.run(CompletableFuture.java:1705)
    at java.base/java.util.concurrent.CompletableFuture$AsyncSupply.exec(CompletableFuture.java:1692)
    at java.base/java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:290)
    at java.base/java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1020)
    at java.base/java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1656)
    at java.base/java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1594)
    at java.base/java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:183)
```

**Comment by HTG-YT** [comment]
Can vouch for this issue on CLion 2021.2.

**Comment by Arion-Kun** [comment]
Can vouch for this issue too, 2021.2

**Comment by GreemDev** [comment]
Occurred in 2021.2 Rider, W10 20H2.

**Comment by InfRandomness** [comment]
A fix has been merged into the development branch of the plugin, you can acquire it by switching to the EAP version of the plugin using those instructions : 
0) Open your IDE
1) Go to Settings > Plugins
2) Click on the gear icon at the top
3) Select Manage Plugin Repositories...
4) Click add in the top right corner
5) Paste this link https://plugins.jetbrains.com/plugins/eap/10233
6) Close the Settings and click on Help > Check for updates...

closing;

**Comment by Almighty-Alpaca** [comment]
I'm reopening this until the fix is on stable for better discoverability

**Comment by Escarnope** [comment]
I've done what InfRandomness said, and it installed no problem

However, it doesn't actually do anything.

I have 1.6.1+14, thought about making my own ticket, but it's related to this one so figured I'd just post here

**Comment by Almighty-Alpaca** [comment]
The fix for this is in v1.7.0 which is just being released should be available in the plugin repository tomorrow.
